### PR TITLE
S2 game screen

### PR DESCRIPTION
**Pull request by MrLukey**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                <h2>How to play:</h2>
+                <ul>
+                    <li>Press 'Start Game' to begin</li>
+                    <li>Read the quote displayed, enter your guess and click 'Submit'</li>
+                    <li>Pressing 'Hint' reveals 3 possible answers and the year the film was released</li>
+                    <li>Click 'Next' to move to the next questions</li>
+                    <li>Once a game is finished, click 'Start Again' to start a new game</li>
+                </ul>
+            </div>
+            <button id="start_button">Start Game</button>
```
Comment (charliecog) [inline index.html]: remove start button as it is considered 'unfinished functionality' at this point. Should be brought in on the story that it becomes functional

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<html lang="en-gb">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="guess_the_film.css">
+    <title>Guess the Film</title>
+</head>
+<body>
+    <main>
+        <section id="start_screen">
```
Comment (charliecog) [inline index.html:11]: missing your background image

File: guess_the_film.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+main {
+    display: flex;
+    flex-flow: row nowrap;
```
Comment (charliecog) [inline guess_the_film.css]: is this doing anything?

File: guess_the_film.css [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    justify-content: center;
+}
+
+section {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
```
Comment (charliecog) [inline guess_the_film.css]: is this doing anything?

**Review by charliecog — CHANGES_REQUESTED** [review]
both CSS files should follow the example set by assets and src directory and should live inside a css directory

File: guess_the_film.css (hunk 3 above)
Comment (albertoroldanq) [reply guess_the_film.css]: it puts the section in the center

File: guess_the_film.css (hunk 4 above)
Comment (albertoroldanq) [reply guess_the_film.css]: deleted

File: index.html (hunk 2 above)
Comment (albertoroldanq) [reply index.html:11]: removed in the wireframe

File: index.html (hunk 1 above)
Comment (albertoroldanq) [reply index.html]: done!

**Review by charliecog — APPROVED** [review]

**Pull request merged**